libs/dh/admin/feature-permissions/src/lib/overview/dh-admin-permission-overview.component.ts

```diff
@@ -81,10 +87,12 @@ export class DhAdminPermissionOverviewComponent implements OnInit, OnDestroy {
         this.error = error;
       },
     });
+    this.refresh();
```

Comment:
question: is this call necessary? I probably necessary on refresh but not on initialization. 🤔

Reply:
Fixed